src/main/java/uk/gov/companieshouse/overseasentitiesapi/controller/OverseasEntitiesController.java

```diff
@@ -66,7 +67,8 @@ public ResponseEntity<Object> createNewSubmission(
                 var validationErrors = overseasEntitySubmissionDtoValidator.validate(overseasEntitySubmissionDto, new Errors(), requestId);
 
                 if (validationErrors.hasErrors()) {
-                    ApiLogger.errorContext(requestId, "Validation errors : " + validationErrors, new Exception());
+
+                    ApiLogger.errorContext(requestId, "Validation errors : " + convertErrorsToString(validationErrors), null);
```

Comment:
can you add a unit test to check the errors are passed to the logger? have a look at https://www.baeldung.com/mockito-mock-static-methods#mocking-a-static-method-with-arguments for mocking a static call and maybe use an argument captor to capture the message

Reply:
I've not heard of unit tests for error logging before but have managed to add something. However, the test is misreporting the result:

Wanted but not invoked:
ApiLogger.class.errorContext(
    <any>,
    "Validation errors : {"errs":[{"error":"Any","location":"Any","locationType":"request-body","type":"ch:validation"}]}",
    null
);
 When I debug I can see it being invoked so I am not sure what problem it is picking up on. I have committed what I have done and will require someone to check this branch out and take a look at it.

Reply:
I'll grab this and take a look now @mattch1 .

Reply:
Change pushed. You just needed to move the actual test execution code into that try block, above the new static verify() check. I also made a couple of other very minor improvements. Please take a look @mattch1 .

Reply:
Just run it locally, looks good, so the position of the try block was the issue?

Reply:
Yes.